iina/Base.lproj/Localizable.strings

```diff
@@ -167,6 +167,12 @@
 "preference.sub_override_level.strip" = "strip";
 "preference.sub_override_level.yes" = "yes";
 
+"preference.sub_override_level.descriptive_text.no" = "No override: render subtitles as specified by the subtitle scripts.";
+"preference.sub_override_level.descriptive_text.force" = "Only allow override using the --sub-ass-* mpv options.";
+"preference.sub_override_level.descriptive_text.scale" = "Allow adjusting scale of ASS subtitles and override using the --sub-ass-* mpv options.";
```

Comment:
It is desirable to keep IINA high level and avoid explicitly mentioning mpv options if we can. This description should build on the description for `yes`. See my comment below on `yes`. Maybe something like:
`Apply the scale setting in addition to the style settings in this section.`

Unfortunately since we only support setting the scale on the quick setting panel this might be a little confusing to users. We may want to add scale support in settings to make this easier for a user to grasp and allow users to be able to change the scale in a way that persists.